src/layouts/Workspace/WorkspacePagesAndFoldersComponents.tsx

```diff
@@ -0,0 +1,233 @@
+import {
```

Comment:
side note - we should probably shift all the components here into `/Workspace/compnents` as their own file rather than putting them in this file